Model/Config.php

```diff
@@ -301,4 +302,19 @@ public function getConfigurationGroupLabel(int $websiteId): ?string
             $websiteId
         );
     }
+
+    /**
+     * Check if Express Pay buttons are enabled.
+     *
+     * @param int $websiteId
+     * @return bool
+     */
+    public function isExpressPayEnabled(int $websiteId): bool
+    {
+        return (bool)$this->scopeConfig->getValue(
```

Comment:
Since you're retrieving a boolean value here, you _should_ use `isSetFlag()` to get the value rather than using `getValue()` and casting the result to boolean.

Reply:
Good call,  updated.